google-cloud-examples/src/main/java/com/google/cloud/examples/bigtable/InstanceAdmin.java

```
@@ -0,0 +1,223 @@
+    this.instanceId = instanceId;
+    this.clusterId = clusterId;
+
+    // [START connecting_to_bigtable]
+    // Creates the settings to configure a bigtable instance admin client.
+    BigtableInstanceAdminSettings instanceAdminSettings =
+        BigtableInstanceAdminSettings.newBuilder()
+            .setProjectName(ProjectName.of(projectId))
+            .build();
+
+    // Creates a bigtable instance admin client.
+    adminClient = BigtableInstanceAdminClient.create(instanceAdminSettings);
+    // [END connecting_to_bigtable]
+  }
+
+  public void run() {
+    createProdInstance();
+    listInstances();
+    getInstance();
+    listClusters();
+    addCluster();
+    deleteCluster();
+    deleteInstance();
+    adminClient.close();
+  }
+
+  /** Demonstrates how to create a Production instance within a provided project. */
+  public void createProdInstance() {
+    // Checks if instance exists, creates instance if does not exists.
+    if (!adminClient.exists(instanceId)) {
+      System.out.println("Instance does not exist, creating a PRODUCTION instance");
+      // [START bigtable_create_prod_instance]
+      // Creates a Production Instance with the ID "ssd-instance",
+      // cluster id "ssd-cluster", 3 nodes and location "us-central1-f".
+      CreateInstanceRequest createInstanceRequest =
+          CreateInstanceRequest.of(instanceId)
+              .addCluster(clusterId, "us-central1-f", 3, StorageType.SSD)
+              .setType(Type.PRODUCTION)
+              .addLabel("example", "instance_admin");
+      // Creates a production instance with the given request.
+      try {
+        Instance instance = adminClient.createInstance(createInstanceRequest);
+        System.out.printf("PRODUCTION type instance %s created successfully%n", instance.getId());
+      } catch (AlreadyExistsException e) {
+        System.err.println("Failed to create instance, already exists: " + e.getMessage());
+        System.exit(0);
```

Please re-throw the exception instead